src/z3c/saconfig/README.rst

```diff
@@ -141,12 +141,13 @@ When a new engine is created by an ``EngineFactory``, an
 ``IEngineCreatedEvent`` is fired. This event has an attribute
 ``engine`` that contains the engine that was just created::
 
+  >>> import six
   >>> from z3c.saconfig.interfaces import IEngineCreatedEvent
   >>> @component.adapter(IEngineCreatedEvent)
   ... def createdHandler(event):
-  ...     print "created engine"
-  ...     print "args:", event.engine_args
-  ...     print "kw:", event.engine_kw
+  ...     six.print_("created engine")
+  ...     six.print_("args:", event.engine_args)
+  ...     six.print_("kw:", event.engine_kw)
```

Comment:
There's a way of using `from __future__ import print_statement` in doctests, then you could use `print()`.

(Or if you refrain from using multiple args, you can always do `print("args: {}".format(event.engine_args))` etc. and it'll work on both major Python versions without future imports.)

Reply:
(There are two ways of using print_function in doctests: you can just write it out in the doctest itself like [zc.relation does](https://github.com/zopefoundation/zc.relation/blob/master/src/zc/relation/README.rst), or you can stuff it into globs like [zope.mimetype does](https://github.com/zopefoundation/zope.mimetype/blob/master/src/zope/mimetype/tests/test_doctests.py#L60).)